src/python/library/tritonclient/http/__init__.py

```diff
@@ -1732,6 +1835,59 @@ def read(self, length=-1):
                             output['name']] = buffer_index
                         buffer_index = buffer_index + this_data_size
 
+    @classmethod
+    def from_response_body(cls,
+                           response_body,
+                           verbose=False,
+                           header_length=None,
+                           content_encoding=None):
+        """A class method to construct InferResult object
+        from a given 'response_body'.
+
+        Parameters
+        ----------
+        response_body : bytes
```

Comment:
You missed the first param -> cls here

Reply:
It is similar to `self`, we don't document those as they just refer to the scope of the function.